test/backend_test.in.cpp

```diff
@@ -4764,6 +4766,30 @@ NGRAPH_TEST(${BACKEND_NAME}, reverse_sequence_n4d2c3h2w2)
     EXPECT_EQ(read_vector<int>(result), expected);
 }
 
+NGRAPH_TEST(${BACKEND_NAME}, generate_mask)
+{
+    Shape scalar{};
+    Shape result_shape{1, 20};
+    auto training = op::Constant::create(element::f32, Shape{}, {1});
+    auto gen_mask = make_shared<op::GenerateMask>(training, result_shape, element::f32, 777, 0.5);
+    auto gen_mask2 = make_shared<op::GenerateMask>(training, result_shape, element::f32, 777, 0.5);
+    auto f = make_shared<Function>(NodeVector{gen_mask, gen_mask2}, op::ParameterVector{});
+
+    auto backend = runtime::Backend::create("${BACKEND_NAME}");
+
+    auto result_tv1 = backend->create_tensor<float>(result_shape);
+    auto result_tv2 = backend->create_tensor<float>(result_shape);
+    backend->call_with_validate(f, {result_tv1, result_tv2}, {});
+    auto result1 = read_vector<float>(result_tv1);
+    auto result2 = read_vector<float>(result_tv2);
+    ASSERT_EQ(result1, result2);
```

Comment:
Should also check that `result1` and `result2` (and `result1_2`, `result2_2` below) consist entirely of zeros and ones.